tests/test-runner/parser.js

```diff
@@ -25,12 +25,12 @@ Parser.prototype.Parse = function(file, output) {
         if(contains(line, timeText)) {
             parsedOutput.time = line.replace(timeText,"");
         }
-        else if(contains(line, "assertions") && contains(line, "failure") && contains(line, "skipped")) {
+        else if(contains(line, "assertion") && contains(line, "failure") && contains(line, "skipped")) {

```

Comment:
There's a spec file with just one test, and this was failing to parse its results. Made some small tweaks here so it works properly.